nasa-gibs/worldview
# WV 2679 layer picker menu header bug

## Description

This PR fixes behavior in the "Add Layer" workflow where the header scrolls off-screen. The header should remain at the top of the screen while scrolling through the available layers.

## How To Test

1. `git checkout WV-2679-layer-picker-menu-header-bug`
2. `npm ci`
3. `npm run watch`
4. Got to `http://localhost:3000`
5. Use dev tools to emulate a mobile device.
5. Click on `+Add Layers`
6. Scroll through the available layers.
7. Confirm that the category header remains at the top of the screen.
